source/module_io/input.cpp

```diff
@@ -2706,6 +2706,18 @@ void Input::Default_2(void) // jiyy add 2019-08-04
             lcao_ecut = ecutwfc;
             ModuleBase::GlobalFunc::AUTO_SET("lcao_ecut", ecutwfc);
         }
+
+        // if calculation is get_wf, function source/module_basis/module_pw/pw_basis_k_big.h/distribute_r()
+        // will calculate nbx/nby/nbz by divide nx/ny/nz by bx/by/bz, so bx/by/bz should not be 0
+        if (calculation == "get_wf")
```

Comment:
Thanks for your contribution, it can solve the related issue.
To be honest, I think it is not reasonable to use bx/by/bz in get_wf calculation.